### ISSUE TITLE
CUDA Error Out of Memory - Quality issues?

### ISSUE THREAD

**Issue by lfernandes** [AlexeyAB/darknet#7939]
Hello, 

While generating a custom model, with high performance GPU - like NVIDIA A100 / V100 - we can use a combination of `batch` and `subdivisions` as following:

```
# Training
batch=64
subdivisions=16
```

But when using a simpler GPU - like RTX3070 - it get CUDA out of memory error, then we need to adjust `subdivisions` to `64`.

My question is, does it affect the training model quality?

Thank you in advance!

**Comment by lsd1994** [comment]
Yes, model will be more stable when batch/subdivisions is greater, especially for big dataset.
